packages/qwik-city/middleware/node/http.ts

```diff
@@ -86,16 +86,14 @@ export async function fromNodeHttp(
         start(controller) {
           res.on('close', () => controller.error());
         },
-        write(chunk) {
-          return new Promise((resolve, reject) =>
-            res.write(chunk, (cb) => {
-              if (cb) {
-                reject(cb);
-              } else {
-                resolve();
-              }
-            })
-          );
+        write(chunk, controller) {
+          res.write(chunk, (error) => {
+            if (error) {
+              // FIXME: Ideally, we would like to inform the writer that this was an error.
```

Comment:
this does not seem right

Reply:
i dont want to merge something that looks WIP, what do we do here?

Reply:
The original code, before my previous PR.

```
write(chunk) {
   res.write(chunk);
}
```

We were basically throwing away errors from res.write(). Now, if we pass the error to the controller, Node will instead die because there is a floating promise somewhere that isn't taking care of errors. 

In any case, I wouldn't say it's a WIP. This fix is to make sure Node does not die by going back to how it was before. The note I made there is because people reading the code should be aware that errors disappear there. We can go back to it just being the above three lines if you like, but it would probably be a good idea to make a note somewhere that we are very much aware of what's going on there.